lib/elixir/src/elixir_erl.erl

```diff
@@ -549,7 +549,7 @@ signature_to_binary(Module, '__struct__', []) ->
 signature_to_binary(_, Name, Signature) ->
   'Elixir.Macro':to_string({Name, [], Signature}).
 
-checker_chunk(#{definitions := Definitions, deprecated := Deprecated}, NoWarnUndefined) ->
+checker_chunk(#{definitions := Definitions, deprecated := Deprecated, is_behaviour := IsBehaviour}, NoWarnUndefined) ->
```

Comment:
I would like to avoid adding new fields to the map. The map is stored on the .beam, so it becomes somewhat public, and every addition increases the size. I would compute it here.

Reply:
Not sure how we can avoid this. We need to load all exports from the module map in the parallel checker. The only way I know of determining if a module is a behaviour is by looking in the module ETS tables, and at this point the tables are now longer available.

Reply:
I see, thanks for clarifying! Only one comment left.